Steam Web Integration.user.js

```diff
@@ -317,13 +318,13 @@ function refreshBundles(callback) {
 function doApp(elem, wishlist, ownedApps, ignoredApps, decommissioned, limited, cards, bundles, dlc, lcs, dlcs, dlclcs, llcs, clcs, blcs) {
     $(elem).addClass(`swi`);
 
-    const attr = settings.attributes.find((a) => /apps?\/[0-9]+/g.test($(elem).attr(a)));
+    const attr = settings.attributes.find((a) => /a(pps)?\/[0-9]+/g.test($(elem).attr(a)));

```

Comment:
`a(pps)?` will only match `a` and `apps`, but we also want to match `app`. I think `a(pps?)?` should work for this.